jobs/e2e-test-cypress.groovy

```diff
@@ -73,66 +71,56 @@ Defaults to GIT_REVISION.""",
 """,
    ["all", "deploy", "nightly"]
 
-
 ).apply()
 
 def _setupWebapp() {
-   GIT_SHA1 = kaGit.resolveCommitish("git@github.com:Khan/webapp",
-                                     params.GIT_REVISION);
-   
-   kaGit.safeSyncToOrigin("git@github.com:Khan/webapp", GIT_SHA1);
-
-   dir("webapp") {
-      sh("make clean_pyc");
+   kaGit.safeSyncToOrigin("git@github.com:Khan/webapp", params.CYPRESS_GIT_REVISION);
 
+   dir("webapp/services/static") {
+      sh("yarn install")
```

Comment:
Better would be to do:
```
make -C webapp static_npm_deps
```
in case there's more to do for getting the deps set up right.  But this probably works too.

Reply:
Basically, we just need to install `lambdatest-cli` with dependencies inside lambdaconfig file, so we should be good.